README.th.md

```
@@ -0,0 +1,208 @@
+end
+
+-- อินพุต
+game:GetService("UserInputService").InputBegan:Connect(function(input, gameProcessed)
+	if gameProcessed then return end
+	if input.UserInputType == Enum.UserInputType.MouseButton1 then
+		fire()
+	end
+end)
+```
+
+### Blockcast & Spherecast
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)